__tests__/components/benefit_list_test.js

```diff
@@ -60,11 +60,12 @@ describe("BenefitList", () => {
     let BLInstance = shallow(
       <BenefitList {...props} {...reduxData} />
     ).instance();
-    expect(
-      BLInstance.sortBenefits(benefitsFixture, "en", "popularity").map(
-        b => b.id
-      )
-    ).toEqual(["benefit_2", "benefit_1", "benefit_0", "benefit_3"]);
+    expect(BLInstance.sortBenefits(benefitsFixture).map(b => b.id)).toEqual([
+      "benefit_2",
```

Comment:
This change was mostly the prettifier, but I did remove the unused arguments that were getting passed into `sortBenefits`.